test/pdf/test_helper.rb

```diff
@@ -42,7 +42,12 @@ def dir
   end
 
   def match_pdf?
-    system("diff-pdf --mark-differences --output-diff=#{diff_pdf} #{expect_pdf} #{actual_pdf}")
+    opts = [
+      '--mark-differences',
+      # Allow for small differences that cannot be seen
+      '--channel-tolerance=40'
```

Comment:
A small difference in Prawn2.4 causes the pdf test to fail, but it is not a problem since the difference is not visible.

Here, i have added a setting to allow similar invisible differences in the pdf test.